cli/docker/environment_source.go

```diff
@@ -0,0 +1,56 @@
+package docker
+
+import (
+	"os"
+	"strings"
+
+	"github.com/bitrise-io/bitrise/log"
+)
+
+// implementing env.EnvironmentSource
+type DockerEnvironmentSource struct {
+	Logger log.Logger
+}
+
+func (des *DockerEnvironmentSource) GetEnvironment() map[string]string {
+	passthroughEnvsList := strings.Split(os.Getenv("BITRISE_DOCKER_PASSTHROUGH_ENVS"), ",")
+	passthroughEnvsList = append(passthroughEnvsList, "PATH", "PR", "ENVMAN_ENVSTORE_PATH")
```

Comment:
Please explain the reasoning or future us won't have any idea why it works this way 😆

Reply:
I just want future me and future you to understand what's going on when looking at this codebase in 1 year